md/Introduction.mdx

```diff
@@ -3,9 +3,9 @@ At Gloo we believe that every person is on a journey of growth within their life
 
 This is at the core of what Gloo is about and, as such, we are called to ensure that we are supporting every employee on their personal and professional growth journey.
 
-This document should serve as your guide as you approach professional, career and personal growth as part of the Gloo engineering team. It not only serves as a framework that you may apply to your whole career, but also will give you a practical understanding of how to progress here at Gloo and how your progression will affect your compensation, title and the expectations that your team will have of you.
+This document should serve as your guide as you approach professional, career and personal growth as part of the Gloo engineering team. It not only serves as a framework that you may apply to your whole career, but also will give you a practical understanding of how to progress here at Gloo and how your progression will affect your compensation, role and the expectations that your team will have of you.
 
-It is important that we treat this framework as a guide, not a rulebook or checklist.  We should take a common sense approach and use this to enable meaningful conversations between you and your manager.
+We encourage you to own your career here at Gloo and you can expect support from your manager, the team and the company. We treat this framework as a guide, not a rulebook or checklist. We take a common sense approach and use this to enable meaningful conversations between you and your manager.
```

Comment:
We treat this framework as a guide, not a rulebook or checklist. We take a common-sense approach and use this to enable meaningful conversations between you and your manager. We encourage you to own your career growth as you seek your path of advancement here at Gloo. You can expect support from your manager, the team and the company along the way.